tests/data/reference_graphs/quantized/hw/VPU/mobilenet_v2.dot

```diff
@@ -152,123 +152,123 @@ strict digraph  {
 "150 MobileNetV2/adaptive_avg_pool2d" [id=150, scope=MobileNetV2, type=adaptive_avg_pool2d];
 "151 MobileNetV2/Sequential[classifier]/Dropout[0]/dropout" [id=151, scope="MobileNetV2/Sequential[classifier]/Dropout[0]", type=dropout];
 "152 MobileNetV2/Sequential[classifier]/NNCFLinear[1]/linear" [id=152, scope="MobileNetV2/Sequential[classifier]/NNCFLinear[1]", type=linear];
-"/nncf_model_input_0" [color=purple, id=0, label="Quantizer: [B:8 M:A SGN:ANY W:N PC:N] [B:4 M:S SGN:ANY W:N PC:N] ", scope="", type=nncf_model_input];
```

Comment:
Only int8 bitwidth is left after propagation, how does HAWQ choose int4 and int2 now? Is it called in the controller, not after the "experimental" stage of propagation?

Reply:
This is the graph that is built after the model has left `create_compressed_model`, and the information about the possible configurations at algorithm build time is essentially lost, so now this kind of (test) graph only reflects the final, possibly precision-inited quantized graph state.